app/views/staff/invitations/new.html.erb

```diff
@@ -8,6 +8,9 @@
   <% end -%>
 
   <%= f.govuk_text_field :name %>
-
+  <br>
```

Comment:
We don't need the br-ses. The form builder should take care of the spacing for us. We could probably put these in a checkbox group with the title 'Permissions' and get rid of the 'access' and 'permissions' from the two check box labels.

Reply:
The spacing was not correct  before the br tag. I'll put them in a checkbox group and maybe it will be correct without them, then.

<img width="979" alt="Screenshot 2022-11-21 at 08 48 28" src="https://user-images.githubusercontent.com/45995847/203005883-be4d0c85-52f5-42df-9875-5637c0a2b30c.png">